src/ApiFunctions/GetIssuesByProject.ts

```diff
@@ -0,0 +1,93 @@
+export interface Issue {
```

Comment:
all the file name and directory name conventions should be followed. The directory name and file name should be written in `lower case with dashes`  example ApiFunctions to api-functions and GetIssuesByProject to get-issues-by-project etc.